.github/ISSUE_TEMPLATE/enhancement_template.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Enhancement Request
+about: Suggest an idea for Kolibri Studio
+title: "[Title]: Brief description"
+labels: ''
+assignees: ''
+
+---
```

Comment:
Since these have been added to the design system repo, we're seeing some side-effects. I don't know what's supposed to happen, but I don't think it's this:

>  ![image](https://user-images.githubusercontent.com/2367265/109112006-c7b8c080-76ee-11eb-9cb7-4afa7f432ee2.png)

Reply:
Ah - I didn't realize that! I'll take it out here and in KDS. Thanks for pointing it out.

Reply:
Ahh. Interesting. I was using this from [https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository), but I think I don't need to do that with a PR template!

> To automatically set a default issue title, assign the issue to people with read access to the repository, or apply labels to your issue template, enter these details under "Optional additional information." You can also add these details in the issue template with title, labels, or assignees in a YAML frontmatter format.

I'll fix in KDS, as well.